tests/Domain/DonationRepositoryTest.php

```diff
@@ -308,18 +306,19 @@ public function testStripeAmountForCharityWithTipUsingAmex(): void
         // is not included in the core donation amount set by `setAmount()`.
         $donation = $this->getTestDonation('987.65');
         ;
-        $donation->setPsp('stripe');
         $donation->setTipAmount('10.00');
-        $this->getRepo()->deriveFees($donation, 'amex', null);
+        $donation->deriveFees('amex', null);
 
         // £987.65 * 3.2%   = £ 31.60 (to 2 d.p.)
         // Fixed fee        = £  0.20
-        // Total fee        = £ 31.80
+        // Total fee ex vat = £ 31.80
+        // Total fee inc vat = £ 31.80 * 1.2
+        // Total fee inc vat = £ 38.16
         // Amount after fee = £955.85
 
         // Deduct tip + fee.
-        $this->assertEquals(4_180, $donation->getAmountToDeductFractional());
-        $this->assertEquals(95_585, $donation->getAmountForCharityFractional());
+        $this->assertEquals(48_16, $donation->getAmountToDeductFractional());
```

Comment:
total deduction £41.80 to £48.16 here, only because in the old version of the code we didn't apply UK VAT to the fee as it was settings dependant, so applied in prod but not in this test. Now it's applied here, adding £6.36

There's also a formatting change, `48_16` is meant to be a more intuitive way to write a figure in pounds and pence than the version with the underscore before three digits.